build/release-notes/avalanchego.md

```diff
@@ -2,6 +2,20 @@
 
 {% page-ref page="../tutorials/nodes-and-staking/upgrade-your-avalanchego-node.md" %}
 
+## v1.4.12 \([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.4.12)\)
+
+This update is backwards compatible. It is optional, but encouraged.
+
+**X-Chain**
+* Added methods `GetNodeVersion` and `GetNodeIP` to client
```

Comment:
`GetNodeIP` was already in the client, and these are on the info API, not the X-chain.